inst/crunch-test.R

```diff
@@ -2,9 +2,34 @@ library(httptest)
 
 run.integration.tests <- Sys.getenv("INTEGRATION") == "TRUE"
 
-skip_locally <- function (...) {
-    if (grepl("^https?://local\\.", getOption("crunch.api"))) {
-        skip(...)
+
+
+#' Skip tests when they are being run locally
+#'
+#' @param ... message to be passed to `skip()`
+#' @param backend should the following tests be skipped if the backend is a
+#'   local vagrant backend
+#' @param test_env should the following tests be skipped if the environment the
```

Comment:
Because this isn't in the `R/` directory, the roxygen isn't parsed into documentation at all, so with or without `@keywords internal` it won't show up in the docs/pkgdown site regardless. I added it anyway just in case at some point we end up copying and pasting it somewhere where it does get documented.